[2024-05-21T18:18:00.607+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T18:18:00.621+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:17:58.760441+00:00 [queued]>
[2024-05-21T18:18:00.625+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:17:58.760441+00:00 [queued]>
[2024-05-21T18:18:00.625+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-21T18:18:00.633+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-21 18:17:58.760441+00:00
[2024-05-21T18:18:00.639+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=63) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T18:18:00.640+0000] {standard_task_runner.py:63} INFO - Started process 65 to run task
[2024-05-21T18:18:00.640+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-21T18:17:58.760441+00:00', '--job-id', '543', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmp_obo9tns']
[2024-05-21T18:18:00.642+0000] {standard_task_runner.py:91} INFO - Job 543: Subtask create_db
[2024-05-21T18:18:00.668+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-21T18:17:58.760441+00:00 [running]> on host 4010141b380c
[2024-05-21T18:18:00.718+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T18:17:58.760441+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T18:17:58.760441+00:00'
[2024-05-21T18:18:00.718+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T18:18:00.737+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-21T18:18:01.925+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T18:18:01.926+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T18:18:01.933+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240521T181758, start_date=20240521T181800, end_date=20240521T181801
[2024-05-21T18:18:01.975+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T18:18:01.997+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T18:18:01.998+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
